.github/workflows/mypy.yml

```diff
@@ -14,8 +16,8 @@ jobs:
         python-version: 3.7
     - name: Install Dependencies
       run: |
-        pip install netkan/.[test]
+        pip install --user netkan/. --no-warn-script-location --force-reinstall
+        pip install mypy
```

Comment:
Our test dependencies *should* come from within `setup.py`. The reason pip was installing mypy directly was that calling `[test]` will install all the deps plus the test ones. `--force-reinstall` ensures all deps get installed as user, there isn't any need to install as user in this environment.

Reply:
I don't think we've been well-served by the concept of "test dependencies". We need mypy here and nowhere else, and installing it elsewhere caused 10 hours of downtime.

Reply:
Well, it was the manual installation of `mypy` in the container that caused the downtime. We won't be installing our test dependencies in the container for now and if we do, we'll need to think about how best to approach it. For example we could have a separate 'test' stage, that is `FROM production as test` that we could run as an build action

```Dockerfile
FROM production as test
USER root
ADD . /netkan
RUN chown -R netkan:netkan /netkan
WORKDIR /netkan
USER netkan
RUN pip install --user /netkan/.[test]
RUN mypy .
RUN pytest -v
```


```yml
      - name: Build Test
        id: docker_build
        uses: docker/build-push-action@v2
        with:
          context: ./netkan
          file: ./netkan/Dockerfile
          push: false
          target: test
```